tests/template.fmt

```diff
@@ -37,7 +37,8 @@ void test_assert(const char *file, unsigned line,
 
 
 // utility functions for traversals
-int test_count(void *p, lfs_block_t b) {{
+static int __attribute__((used)) test_count(void *p,
+	lfs_block_t b __attribute__((unused))) {{
```

Comment:
Thoughts on using a `(void)b;` statement instead? Would be portable across compilers. (Though I realize this file is already gcc specific). I can update it in a bit if you're not opposed.